### DISCUSSION TITLE
Fedora 38 on the surface pro 7 is wonderful

### DISCUSSION THREAD

**Discussion by Jeremycouturier** [linux-surface/linux-surface#1127] [General]
Thanks a lot to the developers of this awesome github project for making linux possible on the surface devices.

With the surface kernel, the linux experience is really smooth and it even feels like fedora is more legitimate than windows on the pro 7.

That is all this post is about. Simply say thank you

Jérémy

**Comment by Cyborgscode** [comment]
F37 on an old ATOM tablet... SAME FEELING :)

**Comment by benjamin051000** [comment]
Yup. I have an SP5 and I'm not sure what, but after updating today to Fedora 38, the wake from suspend with the keyboard cover and the power button is SO much more responsive than before... Thank you so much devs!
